rust/kernel/str.rs

```diff
@@ -595,3 +602,22 @@ impl Deref for CString {
 macro_rules! fmt {
     ($($f:tt)*) => ( core::format_args!($($f)*) )
 }
+
+/// Trim the whitspaces
```

Comment:
```suggestion
/// Trim the whitespaces
```
I think this is missing an e